README.md

```diff
@@ -14,7 +14,7 @@ We welcome contributions to this repository. Please open a pull request or an is
-| scMulan                  | 📄(RECOMB)[Bian et al. 2024](https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_57)                                                                       | [🔍Github](https://github.com/SuperBianC/scMulan/tree/main)            | scRNA-Seq                                                | 10M / cross-tissue, human ([hECA](https://www.sciencedirect.com/science/article/pii/S2589004222005892))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Not specified                                                                                                                                                                                          | Decoder                                              | Conditional cell generation                                                                                                 | cell type annotation, cell metadata annotation (both also used in training)                                                                                    | Batch integration                                                                                  |
```

Comment:
Great to see your PR!
I don't see scPRINT, only removed and re-added scMulan. I suppose that's a mistake